executor/index_join.go

```diff
@@ -73,7 +73,7 @@ type IndexLookUpJoin struct {
 
 // Open implements the Executor Open interface.
 func (e *IndexLookUpJoin) Open() error {
-	e.curBatchSize = 32
+	e.curBatchSize = e.ctx.GetSessionVars().InitialIndexJoinBatchSize
```

Comment:
`IndexLookUpJoin` now is never used, you should modify `NewIndexLookUpJoin` instead. See `func (b *executorBuilder) buildIndexLookUpJoin(v *plan.PhysicalIndexJoin) Executor {` for more detail.

Reply:
This is a tmp fix in release-1.0. There's no NewIndexLookUpJoin or outerWorker...